src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -4034,6 +4046,30 @@ const std::string ApplicationManagerImpl::DirectoryTypeToString(
   return "Unknown";
 }
 
+bool ApplicationManagerImpl::AddLockScreenDismissalWarningToMessage(
+    SmartObjectSPtr notification_so, uint32_t app_id) {
+  auto application = ApplicationManagerImpl::application(app_id);
+
+  if (!application) {
+    return false;
+  }
+
+  const auto mobile_language =
+      MessageHelper::MobileLanguageToString(application->ui_language());
```

Comment:
@IGapchuk  I have already seen this code. Duplication?

Reply:
@LuxoftAKutsan Fixed in 048fe1415fce72016340c817c56367238ee052f6